rc/.eslintrc.json

```diff
@@ -277,7 +312,10 @@
             2,
             "always"
         ],
-        "space-before-function-paren": [0, "always"],
+        "space-before-function-paren": [
+            2,
+            "always"
```

Comment:
I think this is a no-op anyways since we require functions to be named, but yes, you are correct, this should be "never"